.github/workflows/pythonbuild.yml

```diff
@@ -39,9 +39,11 @@ jobs:
         python -m pip install --upgrade pip==21.2.4 setuptools wheel
         make setup${{ matrix.spark-version-suffix }}
         pip freeze
+    - name: Test FlyteSchema compatibility
```

Comment:
can revert this, either way, doesn't really matter.  i think i like not running the compat stuff with coverage.